Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
I see your point about consistency.  Docker isn't perfect here either because we're installing using apt-get, which just gets the latest version of everything, but it's better than relying on the local environment.

The big advantage to not using docker for daily development is that people in resource constrained environments can build more easily/cheaply because they don't have to install and run a docker machine.